kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkTask.java

```diff
@@ -45,7 +45,7 @@
 import io.confluent.connect.storage.partitioner.PartitionerConfig;
 
 public class S3SinkTask extends SinkTask {
-  private static final Logger log = LoggerFactory.getLogger(S3SinkTask.class);
+  public static final Logger log = LoggerFactory.getLogger(S3SinkTask.class);
```

Comment:
Let's keep this as private and create a separate logger for V2 S3 Sink Task class.